### DISCUSSION TITLE
More languages

### DISCUSSION THREAD

**Discussion by ksun2010** [Hankypoo7/The-blooket-hacks#104] [Q&A]
Maybe you could have hacks with more different languages, like css, html, or python.  Like a more variety.

**Comment by ksun2010** [comment]
so if people for some reason can't use js, then they could use python or any other languages.

**Comment by VillainsRule** [comment]
Hello! Welcome to our Blooket Hacks repository!

Running through your list, you mentioned CSS, HTML, or Python.
CSS is how a website looks, the style + theme. It is not possible to create a Blooket hack with stylesheets.
HTML is usually the main roots of a site, including text. It is possible to embed Javascript in an HTML site (see [here](https://www.digitalocean.com/community/tutorials/how-to-add-javascript-to-html)).
Python requires a desktop app and might be possible, but this would require further investigation.

Right now, since we have never heard of an issue of being unable to use Javascript while being able to use another language, this is not an issue.

-VillainsRule
